src/components/Tabs/Tabs.js

```diff
@@ -30,22 +31,54 @@ export default class Tabs extends React.Component {
   // Render current active tab content
   renderActiveTabContent() {
     const { children } = this.props;
-    const { activeTabIndex } = this.state;
+    const { activeTabIndex, horizontalLineWidth } = this.state;
     if (children[activeTabIndex]) {
-      return children[activeTabIndex].props.children;
+
+      let newChildren = [];
+      for (let i = 0; i < children[activeTabIndex].props.children.length; i++) {
+        let child = children[activeTabIndex].props.children[i];
+        console.log(child);
+        if (child.props.className && child.props.className.includes("centered_elements")) {
+          let newChild = React.cloneElement(child, {
```

Comment:
Why is this cloning the element? This is going to duplicate the amount of memory used by the tabs. Instead of doing this logic here, recommend either


1. Wrap the children in a div with the property
2. Go through all of the places where Tabs is used and pass the property there. 

Also this method will be called every time the render is called which, at a minimum, will be when the property or state variables change.